move-spec-test/README.md

```diff
@@ -122,20 +106,20 @@ The sample report generated for the above test will look as follows:
 You can try to run the tool using other examples from the `move-mutator`
 tests like:
 ```bash
-./target/release/move spec-test -p third_party/move/tools/move-mutator/tests/move-assets/simple
+./target/release/move-spec-test -p move-mutator/tests/move-assets/simple
 ```
 
 You should see different results for different modules as it depends on the
-quality of the specifications. Some modules, like `Sum`, have good 
+quality of the specifications. Some modules, like `Sum`, have good
 specifications and all mutants are killed, while some others, like `Operators`
 may not and some mutants are still alive.
 
-You can also try the Move Prover testsuite available in the
-`third_party/move/move-prover/tests/sources/` directory.
+You can also try the Move Prover testsuite available in the `aptos-core` repo in
```

Comment:
So maybe we should include the link to the main `aptos-core` directory, since it not present inside our repository anymore.

Reply:
Done